x/bank/spec/README.md

```diff
@@ -22,63 +35,67 @@ supply of all assets used in the application.
 
 This module will be used in the Cosmos Hub.
 
-## Supply
+## 02. Concepts
+
+### Supply
 
 The `supply` module:
 
 - passively tracks the total supply of coins within a chain,
 - provides a pattern for modules to hold/interact with `Coins`, and
 - introduces the invariant check to verify a chain's total supply.
 
-### Total Supply
+#### Total Supply
 
 The total `Supply` of the network is equal to the sum of all coins from the
 account. The total supply is updated every time a `Coin` is minted (eg: as part
 of the inflation mechanism) or burned (eg: due to slashing or if a governance
 proposal is vetoed).
 
-## Module Accounts
+### Module Accounts
 
-The supply module introduces a new type of `auth.Account` which can be used by
-modules to allocate tokens and in special cases mint or burn tokens.  At a base
+The supply module introduces a new type of `auth.AccountI` interface, called `ModuleAccountI`, which can be used by
```

Comment:
Similarly, there isn't a supply module anymore, so we should probably remove references to it as such.